<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-ORI8JR52" ignored="2" name="Test" tests="2" failures="1" timestamp="2020-04-18T18:24:30 IST" time="3.047" errors="0">
  <testcase name="loginAsBankManager" time="0.419" classname="com.w2a.DataDrivenFramework.testCases.BankManagerLoginTest"/>
  <testcase name="addCustomer" time="0.953" classname="com.w2a.DataDrivenFramework.testCases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.w2a.DataDrivenFramework.testCases.AddCustomerTest.addCustomer(AddCustomerTest.java:29)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomer -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="com.w2a.DataDrivenFramework.base.TestBase">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase name="@AfterSuite testDown" time="0.0" classname="com.w2a.DataDrivenFramework.base.TestBase">
    <ignored/>
  </testcase> <!-- @AfterSuite testDown -->
</testsuite> <!-- Test -->
